### PR TITLE
Add Chesnaught and update mine

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
"Passed" Travis

File: mods/essb/moves.js [hunk 1]
```diff
@@ -836,48 +836,44 @@ exports.BattleMovedex = {
...
-				}
-			},
+			selfSwitch: true,
+		},
+		target: "Normal",
+		type: "Ice",
+	},
+	"hyperviperbeam": {
+		accuracy: 100,
+		basePower: 300,
```
Comment (DeathlyPlays) [inline mods/essb/moves.js]: Way too OP e.e

File: mods/essb/scripts.js [hunk 2]
```diff
@@ -143,13 +143,13 @@ exports.BattleScripts = {
...
-				ability: "Ace in the Hole",
-				item: "Power Current",
+				species: "Rotom-frost",
+				ability: "Turbo Mode",
+				item: "Marvelium Z",
 				gender: "M",
-				moves: ['Bug Buzz', 'Glaciate', 'Hydro Pump'],
-				baseSignatureMove: "electrofryer",
-				signatureMove: "Electro-Fryer",
+				moves: ['Hurricane', 'Thunder', 'Volt Switch'],
```
Comment (DeathlyPlays) [inline mods/essb/scripts.js]: Lowercase it

**Review by DeathlyPlays — DISMISSED** [review]

**Comment by ghost** [comment]
@DeathlyPlays

**Pull request merged**